pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -58,7 +58,7 @@ const (
 	// The minimum memory limit allocated to docker container: 150Mi
 	MinDockerMemoryLimit = 150 * 1024 * 1024
 
-	dockerProcessName     = "docker"
+	dockerProcessName     = "dockerd?"
```

Comment:
You are relying on the description of `procfs.Pidof` being incorrect - "Find process(es) with a specified name (exact match)".  I filed #41247